sos/report/plugins/bmc.py

```
@@ -0,0 +1,238 @@
+            return
+
+        # Create temporary .netrc file to avoid credentials in ps output
+        netrc_fd, netrc_path = tempfile.mkstemp(prefix='bmc_netrc_')
+        try:
+            with os.fdopen(netrc_fd, 'w') as netrc_file:
+                netrc_file.write(f"machine {bmc_ip}\n")
+                netrc_file.write(f"login {bmc_user}\n")
+                netrc_file.write(f"password {bmc_password}\n")
+            os.chmod(netrc_path, 0o600)
+
+            dump_dir = Path('/tmp/bmc_sos')
```

Ruff (0.14.13)

Probable insecure usage of temporary file or directory: "/tmp/bmc_sos"

(S108)